README.rst

```diff
@@ -3,8 +3,37 @@ djangosnippets.org
 
 This code is used to power the snippet sharing site, djangosnippets.org
 
+Database Setup Using Windows
+-----------------------------------
 
-Development setup
+Download the latest version of PostgreSQL_. Click on the executable to start the installation setup wizard.
+
+Click ``Next``, keeping all the defaults, ensuring that all components are ticked (including pgAdmin 4). Make a note
+of the password you choose for the database superuser (postgres). Select the default port 5432 and the default locale.
+After it’s finished installing, you do not need to launch Stack Builder. Un-tick that box if you are asked, and
+click ``Finish``.
+
+Open the pgAdmin 4 app. Type in the password you made a note of earlier. The following steps can also be taken using
+your windows CLI: Click on ``Servers`` in the Browser pane on the left. Right click on ``PostgreSQL`` and then click
+on ``Create``, then ``Database``. In the Database field which you should now see, type 'djangosnippets', then click
```

Comment:
It might be better to just use powershell or cmd to work with postgres as that will eliminate the dependency on pgAdmin 4.